rust-toolchain

```diff
@@ -0,0 +1 @@
+1.32.0
```

Comment:
I believe at one point, @TheWaWaR voted against adding this file since this is a library. Thoughts?